tests/test_processCcd.py

```diff
@@ -147,29 +147,32 @@ def testProcessCcd(self):
                     # running the test to determine the updated values.
                     expectedPlaces = 7  # Tolerance for numerical comparisons
                     for name, var, val in [
-                        ("bgMean", bgMean, 191.48623786891795),
-                        ("bgStdDev", bgStdDev, 0.23994185672586282),
-                        ("numGoodPix", numGoodPix, 1966606),
-                        ("imMean", imMean, 1.1242456954648634),
-                        ("imStdDev", imStdDev, 85.8129750182329),
+                        ("bgMean", bgMean, 191.48618251407825),
+                        ("bgStdDev", bgStdDev, 0.23993854036946),
+                        ("numGoodPix", numGoodPix, 1966533.00000000000000),
```

Comment:
`numGoodPix` was previously an integer. Did you mean to cast it to a float here consciously?

Reply:
Good catch.  Integer definitely makes sense for a number of pixels.  Will fix.